### PR TITLE
Add info on license renewals

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

File: docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx [hunk 1]
```diff
@@ -30,3 +30,11 @@ Attempts to use unlicensed products will result in an error message and users wi
 ```code
 this Teleport cluster is not licensed for database access, please contact the cluster administrator
 ```
+
+# License Updates and Renewals
+
+If your license is updated for features and/or renewals an updated license file is required.  Download the updated license file
+from [Teleport Customer
```
Comment (alexfornuto) [inline docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx:37]:
```suggestion
from the [Teleport Customer
```

File: docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx [hunk 2]
```diff
@@ -30,3 +30,11 @@ Attempts to use unlicensed products will result in an error message and users wi
...
+Portal](https://dashboard.gravitational.com/web/login) and reinstall to all Teleport Auth Servers.
+After restarting the Teleport Auth service any changes to the license will take impact.
```
Comment (alexfornuto) [inline docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx:39]:
```suggestion
After restarting the Teleport Auth service any changes to the license will take effect.
```

**Review by alexfornuto — APPROVED** [review]
Disabled auto-merge so you can commit suggestions.

File: docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx [hunk 3]
```diff
@@ -30,3 +30,11 @@ Attempts to use unlicensed products will result in an error message and users wi
 ```code
 this Teleport cluster is not licensed for database access, please contact the cluster administrator
 ```
+
+# License Updates and Renewals
```
Comment (ptgott) [inline docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx:34]:
```suggestion
## License updates and renewals
```
We use sentence case for headings and H2s for first-level headings.

File: docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx [hunk 4]
```diff
@@ -30,3 +30,11 @@ Attempts to use unlicensed products will result in an error message and users wi
...
+
+If your license is updated for features and/or renewals an updated license file is required.  Download the updated license file
```
Comment (ptgott) [inline docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx:36]: Maybe we can copy the more detailed license file instructions from the Teleport Enterprise getting started guide (https://goteleport.com/docs/deploy-a-cluster/teleport-enterprise/getting-started/#obtain-your-license-file) into a partial and include these here?

File: docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx [hunk 5]
```diff
@@ -30,3 +30,11 @@ Attempts to use unlicensed products will result in an error message and users wi
...
+from the [Teleport Customer
+Portal](https://dashboard.gravitational.com/web/login) and reinstall to all Teleport Auth Servers.
```
Comment (ptgott) [inline docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx:38]:
```suggestion
Portal](https://dashboard.gravitational.com/web/login) and reinstall to all Teleport Auth Service instances.
```

File: docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx [hunk 6]
```diff
@@ -30,3 +30,11 @@ Attempts to use unlicensed products will result in an error message and users wi
...
+After restarting the Teleport Auth service any changes to the license will take effect.
```
Comment (ptgott) [inline docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx:39]:
```suggestion
After restarting the Teleport Auth Service, any changes to the license will take effect.
```

**Review by ptgott — APPROVED** [review]

**Pull request merged**